src/libraries/System.Net.Security/src/System/Net/Security/NegotiateAuthentication.cs

```diff
@@ -356,21 +356,23 @@ public NegotiateAuthenticationStatusCode UnwrapInPlace(Span<byte> input, out int
         /// Computes the message integrity check of a given message.
         /// </summary>
         /// <param name="message">Input message for MIC calculation.</param>
-        /// <param name="signature">Buffer writer where the MIC is written.</param>
+        /// <param name="signatureWriter">Buffer writer where the MIC is written.</param>
         /// <remarks>
+        /// Implements the GSSAPI GetMIC operation.
+        ///
```

Comment:
If the intention is to have paragraphs here, you need to use the right XML tags. Otherwise the docs will elide whitespace. So `<para>Implements the GSSAPI GetMIC operation.</para>`

Reply:
Interesting. I didn't know that and it's pre-existing problem with the rest of the API remarks in the same file.

I randomly went to check [GetOutgoingBlob](https://learn.microsoft.com/en-us/dotnet/api/system.net.security.negotiateauthentication.getoutgoingblob?view=net-8.0) and the documentation was rendered correctly on MSDN with just the empty line.

Reply:
In that case maybe it's not an issue anymore? It was at some point? Well, we can fix it in the docs if needed.